socorro/cron/jobs/fetch_adi_from_hive.py

```diff
@@ -345,13 +352,37 @@ def run(self, date):
             self.config.logger.info(
                 'Wrote %d rows from doing hive query' % rows_written
             )
-            for transaction in transactions:
-                transaction(
-                    self._database_transaction,
-                    raw_adi_logs_pathname,
-                    target_date
-                )
+
+            self.persist_data(transactions, raw_adi_logs_pathname, target_date)
 
         finally:
             if os.path.isfile(raw_adi_logs_pathname):
                 os.remove(raw_adi_logs_pathname)
+
+    def persist_data(self, transactions, raw_adi_logs_pathname, target_date):
+        for transaction in transactions:
+            transaction(
+                self._database_transaction,
+                raw_adi_logs_pathname,
+                target_date
+            )
+
+
+class DryRunFetchADIFromHiveCronApp(FetchADIFromHiveCronApp):
+    """Dry run version of FetchADIFromHiveCronApp that doesn't persist any data
+
+    """
+    app_name = 'dry-run-fetch-adi-from-hive'
+    app_description = 'Dry Run Fetch ADI From Hive App'
+    app_version = '0.1'
+
+    required_config = Namespace()
```

Comment:
I'm not positive we need this line, but I didn't want to spend 20 minutes verifying one way or the other.